database/src/migrations/release.0.16/biohub.sql

```diff
@@ -675,17 +593,19 @@ COMMENT ON TABLE management_action_type IS 'List of Management Actions.'
 ;
 
 -- 
--- TABLE: no_sample_permit 
+-- TABLE: permit 
 --
 
-CREATE TABLE no_sample_permit(
+CREATE TABLE permit(
```

Comment:
It looks like there is no sampling conducted boolean in this new permit table.  
I think we still need it though? 
Otherwise we won't be able to populate that part of the UI correctly.
Unless there was talk about removing that field?